src/components/meeting.jsx

```diff
@@ -76,7 +91,7 @@ export default class Meeting extends Component {
 								</p>
 								<ul className={classNames('my-0 mt-1', { 'd-none': (!meeting.types || !meeting.types.length) })}>
 									{meeting.types ? meeting.types.map(type => {
-										return(
+										return (
```

Comment:
this is great, let's do this from now on. in fact, any interest in using jsPrettier to manage formatting? it would require a whole PR to transition, but I think once we got it working it might be 💯

Reply:
I agree whole heartedly! I used Black for Python and it is fantastic for readability.